scripts/test/DirectPropertyManater_OldInterfaceXest.py

```diff
@@ -1,330 +0,0 @@
-# Mantid Repository : https://github.com/mantidproject/mantid
```

Comment:
This has been disabled, had typos in the name and not been meaningfully changed since 2015. There's also another test with the same name that is more recent so I think I'm happy with this being removed :+1: